### ISSUE TITLE
508-defect-2 [COGNITION, SCREENREADER]: Heading should read as a single item

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#17515]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** Jennifer

## User Story or Problem Statement

As a screen reader user, I expect to hear a heading read as the number of items it is, generally just a single item, so that I am clearly able to understand what the content and context is. 

## Details

When the heading is read by a screen reader it is announced as nine items, which is confusing because it really is only one item — the heading. The reason for this is that the content within the H2 is separated by a series of items — such as spans or other [insert React term] — are strung together in a heading. VoiceOver reads them as the number of items, which is confusing information for screen reader users. 

## Acceptance Criteria

- [ ] When the H2 is read it is not announced as nine (or any other number more than one) items

## Environment

* Operating System: MacOS
* Browser: Safari
* Screenreading device: VoiceOver
* Server destination: staging

## Steps to Recreate

1. Enter `https://staging.va.gov/find-locations` in browser
1. Start screen reading device listed in Environment
1. Enter search parameters
1. Verify that the H2 for results reads that it is 9 items (as shown in screenshots below)

### Current HTML output (see screenshot)

```html
<h2 id="search-results-subheader" class="vads-u-font-family--sans vads-u-font-weight--normal vads-u-font-size--base vads-u-padding--0p5 vads-u-margin-y--1" tabindex="-1">
  "Results for ""
  <b>VA health</b>
  "",&nbsp;""
  <b>All VA health services</b>
  ""&nbsp;near ""
  <b>Portland, Maine 04101</b>
  """
</h2>
```

### Recommended

```html
<h2 id="search-results-subheader" class="vads-u-font-family--sans vads-u-font-weight--normal vads-u-font-size--base vads-u-padding--0p5 vads-u-margin-y--1" tabindex="-1">
  Results for VA health and All VA health services near Portland, Maine 04101
  </h2>
```

### Alternative

```html
<h2 id="search-results-subheader" class="vads-u-font-family--sans vads-u-font-weight--normal vads-u-font-size--base vads-u-padding--0p5 vads-u-margin-y--1" tabindex="-1">
  Search Results 
  </h2>
<p>For <strong>VA health</strong>, <strong>All VA health services</strong> near <strong>Portland, Maine 04101</strong></p>
```

## WCAG or Vendor Guidance (optional)

* [508/accessibility understandability/cognition](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/accessibility/WCAG-Checklist.md#understandabilitycognition)

## Screenshots or Trace Logs

![Screen Shot 2020-12-15 at 1 42 50 PM (2)](https://user-images.githubusercontent.com/57469/102259655-5fbf5000-3edd-11eb-84c2-6b72ed2e9f86.png)

![Screen Shot 2020-12-15 at 1 43 05 PM](https://user-images.githubusercontent.com/57469/102259659-6057e680-3edd-11eb-8897-01f598c2b1d5.png)

**Comment by mmiddaugh** [comment]
This is being managed as a site wide item in #17515
